jspsych-uil-utils.js

```diff
@@ -182,9 +198,27 @@ var uil = {};
 
         let is_online = isOnline(getProtocol(), getHostname());
 
-        if (access_key === KNOWN_INVALID_ID) {
+        if (!isUUIDFormat(access_key)) {
+            let message =
+                `The access_key ${access_key} is not in a valid format `       +
+                "It should be 5 groups of hexadecimal numbers (0-9 and a-f) "  +
+                "separated by one single '-'."                                 +
+                "The groups should contain 8,4,4,4,12 hexadecimal characters " +
+                "respectively.";
```

Comment:
While this is 100% factual information, I have my doubts about it's ability to convey meaning to an average user.
I would suggest the following:

The access_key ${access_key} is not in a valid format. Please make sure you copied it correctly into your experiment.
It should be 5 groups of characters (0-9 and a-f) with 8, 4, 4, 4, 12 characters per group respectively.

Reply:
Ok will add that.